recipes/biomechanical-analysis-framework/recipe.yaml

```diff
@@ -0,0 +1,93 @@
+context:
+  name: "biomechanical-analysis-framework"
+  cxx_name: "libbiomechanical-analysis-framework"
+  version: "0.1.0"
+
+
+recipe:
+  name: ${{ name }}
```

Comment:
The formatted name here broke staged-recipes conversion.

Reply:
My bad, I am currently commuting, I will do a PR once I am back in front of a laptop.

Reply:
Woops sorry it was the repo_name in the extra section. I fixed it.